src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/TaskListDataGrid.tsx

```diff
@@ -77,7 +77,7 @@ const TaskListDataGrid: React.FC<TaskListDataGridProps> = ({
   const [deadline, setDeadline] = useState(new Date());
   const [priority, setPriority] = useState(TaskPriority.High);
   const [assignees, setAssignees] = useState<UserPreview[]>([]);
-  const [pageSize, setPageSize] = useState(Number(localStorage.getItem(tableRowCount)));
```

Comment:
So you shouldn't change this here, bc this doesn't save whatever choice they choose in localstorage like it did before. Instead you should change what this item is set to in localStorage to '25' when it hasn't been set yet in TaskListTabPanel